pkgs/games/openxray/default.nix

```diff
@@ -35,25 +35,29 @@ let
     };
   };
 in stdenv.mkDerivation rec {
-  pname = "OpenXRay";
+  pname = "openxray";
   inherit version src;
 
-  hardeningDisable = [ "format" ];
-  cmakeFlags = [ "-DCMAKE_INCLUDE_PATH=${cryptopp}/include/cryptopp" ];
-
+  # Fixes hardening
   patches = [
     (fetchpatch {
```

Comment:
do you mind adding a comment about what the intent of the patch is doing, and when the patch is likely not to be needed
```
    # does x, y, z. Should be able to be removed after >=X.Y.Z
    (fetchpatch {
```

Reply:
Is this better? I don't know when that PR gets merged and upstream's next version name isn't as predictable as x.y.z.

![Bildschirmfoto von 2021-05-07 09-14-47](https://user-images.githubusercontent.com/23431373/117412155-bd981700-af14-11eb-885d-ed8e7bd9894b.png)